tests/FsUnit.Xunit.Test/typed.shouldBeSmallerThanTests.fs

```diff
@@ -0,0 +1,23 @@
+namespace FsUnit.Typed.Test
+
+open Xunit
+open FsUnitTyped
+
+type ``shouldBeSmallerThan tests``() =
+    [<Fact>]
+    member __.``10 should be less than 11``() =
+        10 |> shouldBeSmallerThan 11
+
+    [<Fact>]
+    member __.``10 should not be less than 10``() =
+        (fun () -> 10 |> shouldBeSmallerThan 10)
+        |> shouldFail<System.NullReferenceException>
```

Comment:
`MatchException` is thrown instead of `NullReferenceException `

Reply:
Yep, I've literally just discovered this - having thought I was unable to `dotnet restore`, I had no way to check this. Now, of course, I have discovered `dotnet paket restore`.